cargo-geiger/src/cli.rs

```diff
@@ -11,82 +11,19 @@
 // TODO: Consider making this a lib.rs (again) and expose a full API, excluding
 // only the terminal output..? That API would be dependent on cargo.
 
-use cargo::CliError;
-use crate::format::Pattern;
-use crate::Args;
-use crate::graph::{Graph, Node};
-use cargo::core::compiler::CompileMode;
-use cargo::core::compiler::Executor;
-use cargo::core::compiler::Unit;
-use cargo::core::dependency::DepKind;
-use cargo::core::manifest::TargetKind;
 use cargo::core::package::PackageSet;
 use cargo::core::registry::PackageRegistry;
 use cargo::core::resolver::ResolveOpts;
-use cargo::core::shell::Verbosity;
-use cargo::core::InternedString;
-use cargo::core::Target;
 use cargo::core::{Package, PackageId, PackageIdSpec, Resolve, Workspace};
 use cargo::ops;
-use cargo::ops::CleanOptions;
-use cargo::ops::CompileOptions;
-use cargo::util::paths;
-use cargo::util::ProcessBuilder;
 use cargo::util::{self, important_paths, CargoResult};
-use cargo::CliResult;
 use cargo::Config;
 use cargo_platform::Cfg;
-use colored::Colorize;
-use geiger::find_unsafe_in_file;
-use geiger::Count;
-use geiger::CounterBlock;
-use geiger::IncludeTests;
-use geiger::RsFileMetrics;
-use petgraph::EdgeDirection;
-use petgraph::visit::EdgeRef;
-use std::collections::{HashMap, HashSet};
-use std::ffi::OsString;
-use std::io;
-use std::path::Path;
 use std::path::PathBuf;
 use std::str::{self, FromStr};
-use std::sync::Arc;
-use std::sync::Mutex;
-use walkdir::DirEntry;
-use walkdir::WalkDir;
 
 // ---------- BEGIN: Public items ----------
```

Comment:
I think we can remove these comments now when modules are much smaller. I haven't seen similar comment sections in other Rust projects so unless you insist to keep them let's remove them all.

Reply:
I think that's a good shout, I had kept them in to keep a consistent style, but am happy to take them out.